src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Client/CommitmentsApiClient.cs

```diff
@@ -185,7 +185,12 @@ public Task<string> SecureEmployerCheck()
 
         public Task<string> SecureProviderCheck()
         {
-            return _client.Get("api/test/provider");  
+            return _client.Get("api/test/provider");
+        }
+
+        public Task StopApprenticeship(long apprenticeshipId, StopApprenticeshipRequest request, CancellationToken cancellationToken)
+        {
+            return _client.PostAsJson($"api/apprenticeships/{apprenticeshipId}/stop-apprenticeship", request, cancellationToken);
```

Comment:
I might have called this merely `stop` rather than `stop-apprenticeship`, since the action is against an `apprenticeship` resource so it's pretty clear (also, apprenticeships are the only thing that has a concept of "stopping"). This is pretty pedantic though and would probably need renaming all the way down, which would be pretty tedious.

Reply:
I've renamed the endpoint to be `stop` but left the method name in the controller similar to the others if that works?